mongorm/queryset/Q.py

```diff
@@ -1,18 +1,19 @@
-
+from past.builtins import basestring
+from builtins import object
 class Q(object):
 	def __init__( self, _query=None, **search ):
 		if _query is None:
 			if 'pk' in search:
 				search['id'] = search['pk']
 				del search['pk']
-			
+
 			self.query = search
 		else:
 			self.query = _query
-	
+
 	def toMongo( self, document, forUpdate=False, modifier=None ):
 		newSearch = {}
-		for (name, value) in self.query.iteritems( ):
+		for (name, value) in self.query.items( ):
```

Comment:
This will incur a performance hit on Python 2.x. You might want to consider using `six.iteritems` instead if you want to avoid it.

Reply:
I think the performance impact will be small - most of the larger arrays that were in our mongo documents have been factored out into their own collections (for reasons of mongo/memcached performance).

Reply:
Hmm, I'm surprised it makes that one of the default changes as part of the auto upgrade tool actually... I would have expected it to have gone with option 3 from their site: https://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items

```
# Python 2 and 3: option 3
from future.utils import iteritems
# or
from six import iteritems

for (key, value) in iteritems(heights):
    ...
```

I guess they try default into making it as python3 compatible as possible from the get go, to hint that you should really be moving to py3 sooner rather than later :P

There's a few places that have this change (aka, the [Convert iterX methods to use py3 syntax ](https://github.com/OpenLearningNet/mongorm/pull/44/commits/9af04ec0e61e6dc7d68e3ea3d2aa32cbd2f8fc4a) commit). Do you think it's worth updating them to use that import?